packages/grafana-ui/src/components/uPlot/plugins/TooltipPlugin2.tsx

```diff
@@ -101,7 +114,9 @@ export const TooltipPlugin2 = ({ config, hoverMode, render, clientZoom = false,
 
   const sizeRef = useRef<TooltipContainerSize>();
 
-  const styles = useStyles2(getStyles);
+  const maxWidth = tooltipOptions?.maxTooltipWidth || DEFAULT_TOOLTIP_WIDTH;
+  const maxHeight = tooltipOptions?.maxTooltipHeight || DEFAULT_TOOLTIP_HEIGHT;
```

Comment:
let's use `??` here instead of `||`

Reply:
So if the user enters a value of '0' for a max width, does that mean the tool tip still renders just has no width, and therefore is invisible? At least if we use ||, than a zero value will at least render the default value.

Reply:
oh, yes, it can be zero, good catch 👍🏻